graylog2-server/src/main/java/org/graylog/plugins/sidecar/migrations/V20180212165000_AddDefaultCollectors.java

```diff
@@ -199,8 +201,8 @@ private void ensureFilebeatCollectorsAndConfig() {
                         .append(f(apacheConfigType, """
```

Comment:
The problem here is actually that we are appending multiple Apache configurations for Darwin and Freebsd because we are using the same `StringBuilder`. Each OS configuration should only have one Apache configuration. And I think that's why it slipped through, because we all tested with Linux which configuration was ok.

Reply:
Ok, it should be a single config now. I thought setting two configs was done on purpose but it actually wouldn't make any sense.